cisstMultiTask/mtsCommandWriteReturn.h

```diff
@@ -94,7 +94,7 @@ class CISST_EXPORT mtsCommandWriteReturn: public mtsCommandBase {
     virtual mtsExecutionResult Execute(const mtsGenericObject & argument,
                                        mtsGenericObject & result,
                                        mtsCommandWriteBase * CMN_UNUSED(finishedEventHandler))
-    { return Execute(argument, result, 0); }
+    { return Execute(argument, result); }
```

Comment:
Same question here: Is this the right fix here? It definitely fixes the problematic infinite recursion, but is it delegating to the correct method...?

Reply:
Yes, this is also correct.